pkg/fileutil/fileutil.go

```diff
@@ -57,6 +59,16 @@ func ReadDir(dirpath string) ([]string, error) {
 	return names, nil
 }
 
+// MkdirAllIfNotExist is simliar to os.MkdirAll, but will not return ErrExist
+// if the dir exsits before.
+func MkdirAllIfNotExist(dir string) error {
```

Comment:
Switched to `mkdirAllAlways`. If you still do not feel good about the naming, it probably means that helper func itself is unnecessary . We can just handle the error in main path.